replications/service.go

```diff
@@ -396,3 +401,22 @@ func (s service) populateRemoteHTTPConfig(ctx context.Context, id platform.ID, t
 
 	return nil
 }
+
+func (s service) Open(ctx context.Context) error {
+	if err := s.durableQueueManager.StartReplicationQueues(ctx, s.store); err != nil {
+		return err
+	}
+
+	for id, queue := range s.durableQueueManager.ReplicationQueues() {
+		s.log.Info("Opened replication stream", zap.String("id", id.String()), zap.String("path", queue.Dir()))
```

Comment:
I would expect this log to be part of `StartReplicationQueues`, logged after opening each queue